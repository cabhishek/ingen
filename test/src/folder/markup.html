<!--
{
  "layout": "default",
  "title": "Markup"
}
-->

<p>This page has markup. It's also a test to make sure that files in the regular filesystem get copied to the right place, with the right extension.</p>
