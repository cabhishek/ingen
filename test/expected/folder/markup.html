<!DOCTYPE html>
<html>
  <head>
    <title>InGen</title>
  </head>
  <body>
    <header>InGen</header>

    <main>
<p>This page has markup. It's also a test to make sure that files in the regular filesystem get copied to the right place, with the right extension.</p>
</main>
    <footer>&copy; 2013</footer>

  </body>
</html>
